src/ReactWrapperComponent.jsx

```diff
@@ -32,9 +32,9 @@ export default function createWrapperComponent(node, options = {}) {
       };
     },
 
-    setChildProps(newProps) {
+    setChildProps(newProps, cb = undefined) {
```

Comment:
What are the advantages of doing that here? Specifically for `setState` which is meant to shadow the React API, which does not make this explicit.